docker-compose.yml

```diff
@@ -0,0 +1,54 @@
+# This is the Compose file for command-line services.
+# Anything that doesn't need to be run as part of the main `docker-compose up'
+# command should reside in here and be invoked by a helper script.
+version: "3.7"
+services:
```

Comment:
@coreylafferty this is missing lines from my PR: https://github.com/forumone/gesso-wp/pull/240/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R6-R9